libs/util/dsg-utils/package.json

```diff
@@ -1,3 +1,19 @@
 {
-  "name": "@amplication/dsg-utils"
+  "name": "@amplication/dsg-utils",
+  "version": "1.0.0",
+  "description": "This library supplies common utils for Amplication DSG",
+  "main": "./src/index.js",
+  "author": {
+    "name": "Amplication",
+    "email": "contact@amplication.com",
+    "url": "https://amplication.com"
+  },
+  "dependencies": {
+    "@amplication/code-gen-types": "2.0.37",
```

Comment:
If this will run *inside* the DSG containers, you should probably specify code-gen-types as peerDependency